core/src/main/java/org/kiwix/kiwixmobile/core/search/adapter/SearchViewHolder.kt

```diff
@@ -30,25 +33,31 @@ sealed class SearchViewHolder<in T : SearchListItem>(containerView: View) :
   class RecentSearchViewHolder(
     override val containerView: View,
     private val onClickListener: (SearchListItem) -> Unit,
+    private val onClickListenerNewTab: (SearchListItem) -> Unit,
     private val onLongClickListener: (SearchListItem) -> Unit
   ) : SearchViewHolder<RecentSearchListItem>(containerView) {
     override fun bind(item: RecentSearchListItem) {
       containerView.setOnClickListener { onClickListener(item) }
+      containerView.list_item_search_new_tab_button
```

Comment:
you don't need to prefix these views with containerView, because this ViewHolder extends our `BaseViewHolder` it is also a `LayoutContainer` meaning we can use Kotlin Android Extensions, ie the synthetic view properties

Reply:
@briancherin this hasn't been updated to stop prefixing these views with `containerView.`